### ISSUE TITLE
Import Script fails with database model error

### ISSUE THREAD

**Issue by xlexi** [FuelRats/api.fuelrats.com#18]
```
error:  VersionError: No matching document found.
    at /Users/alex/api.fuelrats.com-fork/node_modules/mongoose/lib/model.js:217:19
    at /Users/alex/api.fuelrats.com-fork/node_modules/mongoose/lib/model.js:170:9
    at handleCallback (/Users/alex/api.fuelrats.com-fork/node_modules/mongoose/node_modules/mongodb/lib/utils.js:96:12)
    at /Users/alex/api.fuelrats.com-fork/node_modules/mongoose/node_modules/mongodb/lib/collection.js:987:5
    at /Users/alex/api.fuelrats.com-fork/node_modules/mongoose/node_modules/mongodb/node_modules/mongodb-core/lib/topologies/server.js:780:13
    at Callbacks.emit (/Users/alex/api.fuelrats.com-fork/node_modules/mongoose/node_modules/mongodb/node_modules/mongodb-core/lib/topologies/server.js:95:3)
    at null.messageHandler (/Users/alex/api.fuelrats.com-fork/node_modules/mongoose/node_modules/mongodb/node_modules/mongodb-core/lib/topologies/server.js:249:23)
    at Socket.<anonymous> (/Users/alex/api.fuelrats.com-fork/node_modules/mongoose/node_modules/mongodb/node_modules/mongodb-core/lib/connection/connection.js:300:20)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at readableAddChunk (_stream_readable.js:146:16)
    at Socket.Readable.push (_stream_readable.js:110:10)
    at TCP.onread (net.js:521:20)
```

**Comment by xlexi** [comment]
Temporarily resolved by 588df89107161e793b32ff2abda99239d50d1f5f better permanent solution required

**Comment by trezy** [comment]
Closing since I've created #24.
